static/metrics-explorer.html

```diff
@@ -172,13 +172,17 @@ <h1 class="myOrg-heading">Metrics Explorer</h1>
                     </div>
                 </div>
                 <hr>
-                <div class="mt-4">
+                <div class="mt-4 d-flex">
+                    <div class="graph-view-container btn" id="alert-from-metrics-container">
+                        <button class="btn alert-from-metrics-btn below-btn-img" id="alert-from-metrics-btn" title="Create Alert"></button>
+                        <label class="label" style="cursor: pointer;">Create Alert</label>
```

Comment:
Should we just add the bell icon to the right of the toggle button as we do on the logs screen?
![Uploading image.png…]()